examples/cmd/simple_run.py

```diff
@@ -0,0 +1,128 @@
+r"""
+mkdir -p records && python -m examples.cmd.simple_run
```

Comment:
Using a model-type running seems meaningless because user should still edit this file and change the texts.